### PR TITLE
fix(lint) unconverted lint fixes & enablement

### PR DESCRIPTION
**Pull request by skrdgraph**
## Problem
unconvert linter missing

## Solution
add unconvert & fix linting issues

### CODE REVIEW

**Comment by skrdgraph** [comment]
@matthewmcneely @mangalaman93 @anurags92 or @harshil-goel review needed here.

**Comment by mangalaman93** [comment]
There is alreqdy a pr here https://github.com/dgraph-io/dgraph/pull/8685

File: graphql/e2e/common/common.go [hunk 1]
```diff
@@ -54,6 +54,7 @@ var (
 	dgraphStateURL  = "http://" + Alpha1HTTP + "/state"
 
 	// this port is used on the host machine to spin up a test HTTP server
+	//nolint:unused
 	lambdaHookServerAddr = ":8888"
```
Comment (hezhizhen) [inline graphql/e2e/common/common.go:58]: It is used in lambda.go:376, so there is no need to add `nolint:unused`

**Comment by mangalaman93** [comment]
Closing in lieu of https://github.com/dgraph-io/dgraph/pull/8685

File: graphql/e2e/common/common.go (hunk 1 above)
Comment (skrdgraph) [reply graphql/e2e/common/common.go:58]: +1 ^ (for some reason when I ran the golangci locally, it flagged a failure for this). I noticed this was being used in another file - which is why I added a `nolint`

But my re-test did not flag a failure here. So lets get your PR in .. thanks for cross checking!

**Pull request closed without merging**